src/frontend/komponenter/Fagsak/BehandlingContainer.tsx

```diff
@@ -26,7 +26,7 @@ const FlexContainer = styled.div`
 
 const VenstremenyContainer = styled.div`
     min-width: 1rem;
-    border-right: 1px solid ${AGray300};
+    border-right: 1px solid ${ABorderDivider};
```

Comment:
Ser ut som ABorderDivider er Grayalpha 300 og ikke Gray 300. Ikke såå stor forskjell på de, men blir det riktig?

Reply:
Jeg mener at jeg kjørte det opp og så på forskjellen. Det viktige her er at vi matcher fargen i bunnen av personlinjen, og jeg mener at den er ABorderSubtle som er lik ABorderDivider
Må kjøre en macos-oppgradering for å sjekke det ut selv, har blitt kastet ut av naisdevice 😅